console/module/about/pom.xml

```diff
@@ -44,6 +44,18 @@
                     </i18nMessagesBundles>
                 </configuration>
             </plugin>
+            <plugin>
+                <!-- GWT 2.4 is limited to Java 6 -->
```

Comment:
Can the `cofiguration` tag be moved to the `pluginManagement` of the `kapua-console` module? Since it repeated also in `kapua-console-core`?